actions/parsing.py

```diff
@@ -75,7 +75,10 @@ def format_isotime_by_grain(isotime, grain=None):
 def parse_duckling_time(
     timeentity: Dict[Text, Any]
 ) -> Optional[Dict[Text, Any]]:
-    timeinfo = timeentity.get("additional_info", {})
+    try:
+        timeinfo = timeentity.get("additional_info", {})
+    except:
```

Comment:
Linting is throwing an error with this line: 
actions/parsing.py:80:5: E722 do not use bare 'except'

Reply:
Ah yes, its was a funky bug based on the time slot being (for reasons I don't understand) prefilled by a string

Reply:
I'll add a specific except